app/assets/javascripts/peek.coffee

```diff
@@ -68,4 +77,5 @@ do ($ = jQuery) ->
 
   $ ->
     if peekEnabled()
+      setPeekEnabledCookie(true)
```

Comment:
We shouldn't set the `peek=true` cookie for every request as it will ignore the user's preference if they've set it to false.